### DISCUSSION TITLE
How to decode gnb_mac.pcap  and ue_mac_nr.pcap by wireshark?

### DISCUSSION THREAD

**Discussion by wmwwimor** [srsran/srsRAN_Project#132] [General Help]
I built a system :srsUE+zmq+srsRAN5G+open5gs，ran this network, according to the manual: 
 https://docs.srsran.com/projects/project/en/latest/user_manuals/source/outputs.html
I tried to decode the data from /tmp/gnb_mac.pcap  and /tmp/ue_mac_nr.pcap by wireshark.
But I have sth wrong when I analyzed a MAC-layer PCAP using Wireshark.

As it said,
1、    Go to Edit->Preferences->Protocols->DLT_USER->Edit and add an entry with DLT=149 and Payload protocol=udp.

 2、  Go to Analyze->Enabled Protocols->MAC-NR and enable mac_nr_udp
I got this result:
...
4	0.300681			UDP	340	48879 → 57005 Len=332 [ILLEGAL CHECKSUM (0)]
5	0.455932			UDP	559	48879 → 57005 Len=551 [ILLEGAL CHECKSUM (0)]
...
So ,what's the "ILLEGAL CHECKSUM"?How can I deal with it?
Thx!!

**Comment by andrepuschmann** [comment]
Can you share the PCAP please?

**Reply by wmwwimor** [reply]
[gnb.log](https://github.com/srsran/srsRAN_Project/files/12070617/gnb.log)
[ue.log](https://github.com/srsran/srsRAN_Project/files/12070629/ue.log)

**Reply by wmwwimor** [reply]
[ue_mac_nr.pcap.txt](https://github.com/srsran/srsRAN_Project/files/12070730/ue_mac_nr.pcap.txt)
[gnb_mac.pcap.txt](https://github.com/srsran/srsRAN_Project/files/12070735/gnb_mac.pcap.txt)

**Reply by wmwwimor** [reply]
Does it mean that the transmitted packets have any issue? Thanks for your reply!

**Reply by pgawlowicz** [reply]
@wmwwimor could you check with the newest srsRAN_Project release?
